tests/tracking/test_tracking.py

```diff
@@ -38,6 +38,28 @@ def test_create_experiment():
         exp_id = tracking.create_experiment("Some random experiment name %d" % random.randint(1, 1e6))
         assert exp_id is not None
 
+def test_get_or_create_experiment():
+    with pytest.raises(TypeError):
+        tracking.get_or_create_experiment()
+
+    with pytest.raises(Exception):
+        tracking.get_or_create_experiment(None)
+
+    with pytest.raises(Exception):
+        tracking.get_or_create_experiment("")
+
+    with temp_directory() as tmp_dir, mock.patch("mlflow.tracking._get_store") as get_store_mock:
```

Comment:
Nit: we can now use the `tracking_uri_mock` to set our tracking URI to a tempdir, e.g.:

```
from tests.projects.utils import tracking_uri_mock
def test_get_or_create_experiment(tracking_uri_mock):
   # Do stuff, the tracking URI will be mocked to point to a temporary directory
```